packages/components/src/components/input-date/input-date.e2e.ts

```diff
@@ -36,6 +37,32 @@ test.describe('kol-input-date', () => {
 			}, TEST_DATE);
 			await expect(page.locator('input')).toHaveValue('04:02');
 		});
+		test('should return Date when initial value is a Date object and trigger onChange', async ({ page }) => {
```

Comment:
Hier würde ich mir eine klarere Bezeichnung wünschen: Wir returnen kein Date, wir übergeben es als Event Detail.

Vorschlag:

```
should dispatch custom 'change'-DOM-event with value as Date object in event detail
```

Den Teil `when initial value is a Date` müssen wir nicht wiederholen, weil es bereits im übergeordneten `test.describe` beschrieben ist.

Der Kommentar gilt auch für die anderen neuen Tests.